packages/forklift-console-plugin/src/utils/resources.ts

```diff
@@ -19,6 +19,10 @@ export const referenceFor = (group: string, version: string, kind: string) =>
  * @param  {K8sResourceCommon} obj
  */
 export const groupVersionKindForObj = (obj: K8sResourceCommon) => {
+  if (!obj?.apiVersion) {
```

Comment:
passing `null`  will cause errors in code that uses this method.
We should either fail fast or provide some default value that would allow the for a graceful failure.